internal/db/schema/migrations/oss/postgres/80/10_auth_method_base_table_updates.up.sql

```diff
@@ -0,0 +1,135 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: BUSL-1.1
+
+begin;
+
+  -- Add create_time, update_time and is_active_public_state to auth_method table.
+  alter table auth_method add column create_time wt_timestamp;
+  alter table auth_method add column update_time wt_timestamp;
+  alter table auth_method add column is_active_public_state boolean not null;
```

Comment:
I don't think you can make this `not null` until after you've done the updates below (unless you also add a default). So normally you'd add this as a boolean here and then do another alter table to set it `not null` after the updates below